template/script.sh.erb

```diff
@@ -5,19 +5,12 @@ modules_to_load, alternate_r_version = context.version.split(':')
 # Load the required environment
 setup_env () {
   module purge
-  # If you want to use Singularity it is recommended you setup a module similar
-  # to the included `example_module` and load it as:
-  #
-  #     module use "/path/to/example_module/modulefiles"
-  #     module load singularity rstudio_singularity/3.4.3
-  #
-  # Bonus points if you install the `example_module` on the system so you don't
-  # need to run `module use`.
-  module load <%= modules_to_load %>
-  <% if alternate_r_version %>
+  module use /users/PZS0002/mrodgers/.local/share/lmodfiles
```

Comment:
I assume after the module is created this line will be removed?

Reply:
Correct